bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ListActiveLedgersCommand.java

```diff
@@ -161,15 +155,17 @@ public void handler(ServerConfiguration bkConf, ActiveLedgerFlags cmdFlags)
       });
     }
 
-    public void printActiveLedgerOnEntryLog(long logId, List<Long> activeLedgers){
-      if (activeLedgers.size() == 0){
+    public void printActiveLedgerOnEntryLog(long logId, EntryLogMetadata entryLogMetadata) {
```

Comment:
Suggest adding a testcase to cover this new method

Reply:
ok